.github/workflows/release.yml

```diff
@@ -0,0 +1,85 @@
+name: Release
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - v*
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go_version:
+        - 1.13.x
+        - 1.14.x
+        - 1.15.x
+        - 1.16.x
```

Comment:
do we have access to `tip`?

Reply:
does not look like, see my comment below; also for example I saw that earlier release builds, before I specified `1.16.x` where done with `1.15`